src/common/darktable.h

```diff
@@ -574,6 +574,18 @@ static inline double dt_get_wtime(void)
   return time.tv_sec - 1290608000 + (1.0 / 1000000.0) * time.tv_usec;
 }
 
+static inline double dt_get_debug_wtime(void)
+{
+  if(!darktable.unmuted)
+    return 0.0;
+  else
+  {
+    struct timeval time;
```

Comment:
Instead of the 3 lines here we should probably call `return dt_get_wtime()` to share the code.

Reply:
right